lib/src/loadable_paginated_list_view.dart

```diff
@@ -35,6 +41,8 @@ class LoadablePaginatedListState<T extends StoreListItem>
           canLoad) {
         viewModel?.loadPage();
       }
+
+      widget.onChangeContentOffset?.call(scrollController.position.pixels);
```

Comment:
It should be unnecessary because we extend the regular list and it already contains this listener.